app/connectors/TariffCommoditiesConnector.scala

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import com.google.inject.Inject
+import config.AppConfig
+import metrics.{ExportsMetrics, MetricIdentifiers}
+import models.responses.TariffCommoditiesResponse
+import play.api.Logger
+import play.api.http.{ContentTypes, HeaderNames, MimeTypes, Status}
+import play.mvc.Http.Status.OK
+import uk.gov.hmrc.http.{HttpClient, _}
+
+import java.net.URL
+import javax.inject.Singleton
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class TariffCommoditiesConnector @Inject()(appConfig: AppConfig, httpClient: HttpClient, metrics: ExportsMetrics)(implicit ec: ExecutionContext) {
```

Comment:
I would rename it to `TariffApiConnector`, as the Tariff API is a provider for several data types, not only for commodities, and it could happen in future that we will use it to retrieve other of those data types.